CHANGELOG.md

```diff
@@ -41,6 +41,11 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Whirlwind](https://github.com/Whirlwind)
   [#7878](https://github.com/CocoaPods/CocoaPods/pull/7878)
 
+* Set Pods group path in user project  
```

Comment:
`Set the path of the Pods group in the Pods project`

Reply:
Is it in the Pods project? not user project?

Reply:
Ah, my bad, there's actually. One in the user project and one in the Pods project. What you have is fine 👍